gui/src/components/DirectoryBrowser.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+  import type { ReadDirResult } from '../lib/api';
+  import HomeSVG from './mono/HomeSVG.svelte';
+  import LeftUpSVG from './mono/LeftUpSVG.svelte';
+  import Textbox from './Textbox.svelte';
+
+  export let dir: ReadDirResult;
+  export let homePath: string;
+  export let handleClick: (path: string) => void;
+  let search: string = '';
+</script>
+
+<div class="toolbar">
+  {#if dir.parent !== null}
+    <button
+      title="Parent directory"
+      on:click={() => handleClick(String(dir.parent?.path))}
+    >
+      <LeftUpSVG />
+    </button>
+  {:else}
+    <button disabled> <LeftUpSVG /> </button>
+  {/if}
+
+  {#if dir.directory.path !== homePath}
+    <button title="Home directory" on:click={() => handleClick(homePath)}>
+      <HomeSVG />
+    </button>
+  {:else}
+    <button disabled> <HomeSVG /> </button>
```

Comment:
Could you do `disabled={dir.directory.path === homePath}`, this way the title is always hooked up & there is less code?